### PR TITLE
chore: fix absolute hrefs, add robot.txt, colorize for archive

### PR DESCRIPTION
**Pull request by wardbell**

### CODE REVIEW

File: public/resources/css/module/_main-nav.scss [hunk 1]
```diff
@@ -30,13 +30,18 @@ $main-nav-background: $blue-700;
 
   h1 {
     margin: 0 ($unit * 9) 0 0;
-    padding: 0px;
+    padding: 10px 0 0 10px;
     height: $main-nav-height;
-    width: 168px;
+    //width: 168px;
```
Comment (IgorMinar) [inline public/resources/css/module/_main-nav.scss]: delete?

File: public/resources/css/module/_main-nav.scss [hunk 2]
```diff
@@ -30,13 +30,18 @@ $main-nav-background: $blue-700;
...
 
-    .md-button, a {
-      background: url('/resources/images/logos/standard/logo-nav.png') 16px 8px no-repeat;
+    a {
+      color: white;
+      text-decoration: none;
+    }
+
+    .md-button {
+      // background: url('/resources/images/logos/standard/logo-nav.png') 16px 8px no-repeat;
```
Comment (IgorMinar) [inline public/resources/css/module/_main-nav.scss]: delete?

File: public/resources/css/module/_main-nav.scss [hunk 3]
```diff
@@ -46,12 +51,12 @@ $main-nav-background: $blue-700;
       padding: 0;
 
       @media only screen and (-webkit-min-device-pixel-ratio: 2), only screen and (min-device-pixel-ratio: 2) {
-        background: url('/resources/images/logos/standard/logo-nav@2x.png') 16px 8px no-repeat;
+        // background: url('/resources/images/logos/standard/logo-nav@2x.png') 16px 8px no-repeat;
```
Comment (IgorMinar) [inline public/resources/css/module/_main-nav.scss]: delete?

**Comment by wardbell** [comment]
Igor ok'd. Merging.

**Pull request merged**